docs/title-and-description.md

```diff
@@ -76,4 +76,55 @@ metadata:
 ```
 
 The above examples will render as rows like the below image:
-![More Markdown Examples](assets/workflow-title-and-description-markdown-complex.png)
+![More Markdown Examples](assets/title-and-description-markdown-complex-workflow.png)
+
+### Cluster Workflow Template
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ClusterWorkflowTemplate
+metadata:
+  name: my-cluster-workflow-template
+  annotations:
+    workflows.argoproj.io/title: '**Test Title**'
+    workflows.argoproj.io/description: |
+      `This is a simple hello world example.`
+      You can also run it in Python: https://couler-proj.github.io/couler/examples/#hello-world
+```
+
+The above examples will render as rows like the below image:
+![More Markdown Examples](assets/title-and-description-markdown-cluster-workflow-template.png)
```

Comment:
The `alt` text should be different for each of these (and, in general, for every image on a page)
```suggestion
![ClusterWorkflowTemplate Example](assets/title-and-description-markdown-cluster-workflow-template.png)
```
also there's only a single example here, so it should be singular "example", not plural "examples"